test/README.md

```diff
@@ -82,13 +98,13 @@ go test -v -tags=e2e -count=1 ./test/e2e
 ```
 
 By default, it will run all applicable tests against the cluster's default
-`ClusterChannelProvisioner`.
+`channel`.
 
-If you want to run tests against other `ClusterChannelProvisioners`, you can
-specify them through `-clusterChannelProvisioners`.
+If you want to run tests against other `channels`, you can
+specify them through `-channels`.
 
 ```bash
-go test -v -tags=e2e -count=1 ./test/e2e -clusterChannelProvisioners=in-memory,gcp-pubsub
```

Comment:
We still don't have a replacement for pubsub. Do you want to keep the pubsub tests running till then?

Reply:
Right, I think I'll hold this PR until `ClusterChannelProvisioner` is completely removed.
Test for `gcp-pubsub` was actually disabled due to test flakiness, now that tests are not flaky anymore, I created https://github.com/knative/eventing/pull/1506 to reenable it.